fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/ValueSetImporter.java

```diff
@@ -62,17 +84,20 @@
 		
 		public void validate() {
 			//check to make sure we are either exporting to file OR importing to a fhir server, but not both
-			if(fileSystemOutputPath != null && measureServerConfigFile != null) {
-				throw new IllegalArgumentException("Parameters [-m, --measure-server] and [-p, --file-system-output-path] cannot both be specified on the same invocation");
+			if(!(fileOutputLocation == OutputLocations.NONE) && measureServerConfigFile != null) {
```

Comment:
Is this just the same as `fileOutputLocation != OutputLocations.NONE`? Or am I misreading this line?

Reply:
yeah, leftover from when it was a different compare, thanks!